sources/data-node/grpc-rest-bindings.yml

```diff
@@ -13,162 +13,159 @@ http:
   rules:
```

Comment:
Why did all simple quote changed to double quote in here? Not that I case much, just curious.

Reply:
That was my editor doing format-on-save, not a conscious choice; I think I set it up to run some sort of yaml linter after some linters in the devops-infra pipeline complained at me. Happy to change it back if you think it's worth it.